src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -0,0 +1,786 @@
+/* dark.css: 👉 light-on-dark */
+:root {
+  --color: rgb(250, 250, 250);
+  --background-color: rgb(5, 5, 5);
+}
+```
+
+In my `style.css`, I then use these variables in the `body { … }` rule.
+As they are defined on the
+[`:root` CSS pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:root)—a
+selector that in HTML represents the `<html>` element
+and is identical to the selector `html`, except that its specificity is
+higher—they cascade down, which serves me for declaring global CSS variables.
+
+```css
+/* style.css */
+:root {
+  color-scheme: light dark;
+}
+
+body {
+  color: var(--color);
+  background-color: var(--background-color);
+}
+```
+
+In the code sample above, you will probably have noticed a property
+[`color-scheme`](https://drafts.csswg.org/css-color-adjust-1/#propdef-color-scheme)
+with the space-separated value `light dark`.
+
+{% Aside 'warning' %}
+  The `color-scheme` property is still [in development](https://crbug.com/925935)
+  and it might not work as advertised, full support in Chrome will come later this year.
+{% endAside %}
+
+This tells the browser which color themes my app supports
+and allows it to activate special variants of the user agent stylesheet,
+which is useful to, for example, let the browser render form fields
+with a dark background and light text, adjust the scrollbars,
+or to enable a theme-aware highlight color.
+The exact details of `color-scheme` are specified in
+[CSS Color Adjustment Module Level&nbsp;1](https://drafts.csswg.org/css-color-adjust-1/).
+
+{% Aside 'note' %}
+  🌒 Read up more on
+  [what `color-scheme` actually does](https://medium.com/dev-channel/what-does-dark-modes-supported-color-schemes-actually-do-69c2eacdfa1d).
+{% endAside %}
+
+Everything else is then just a matter of defining CSS variables
+for things that matter on my site.
+Semantically organizing styles helps a lot when working with dark mode.
+For example, rather than `-⁠-highlight-yellow`, consider calling the variable
+`--accent-color`, as “yellow” may actually not be yellow in dark mode or vice versa.
+Below is an example of some more variables that I use in my example.
+
+```css
+/* dark.css */
+:root {
+  --color: rgb(250, 250, 250);
+  --background-color: rgb(5, 5, 5);
+  --link-color: rgb(0, 188, 212);
+  --main-headline-color: rgb(233, 30, 99);
+  --accent-background-color: rgb(0, 188, 212);
+  --accent-color: rgb(5, 5, 5);
+}
+```
+
+```css
+/* light.css */
+:root {
+  --color: rgb(5, 5, 5);
+  --background-color: rgb(250, 250, 250);
+  --link-color: rgb(0, 0, 238);
+  --main-headline-color: rgb(0, 0, 192);
+  --accent-background-color: rgb(0, 0, 238);
+  --accent-color: rgb(250, 250, 250);
+}
+```
+
+### Full example
+
+In the following [Glitch](https://dark-mode-baseline.glitch.me/) embed,
+you can see the complete example that puts the concepts from above into practice.
+Try toggling dark mode in your particular [operating system’s settings](#activating-dark-mode-in-the-operating-system)
```

Comment:
I initially did this on my mac and didn't see any change. It was only after reading further down that I saw that it's only in Chrome 76. I would suggest pulling that paragraph up before this one and possibly using the feature detection snippet in the glitch itself to show some text to the user saying if their browser either supports or does not support dark mode.

Reply:
I have moved the compatibility paragraph up now. https://github.com/GoogleChrome/web.dev/pull/1021/commits/361af599746d6b691d1e0cd24048bae28f6952b5

Reply:
…and added the feature detection script to the Glitches.